controllers/dependencies.go

```diff
@@ -72,34 +75,67 @@ func (r *OperatorPipelineReconciler) reconcilePipelineDependencies(meta metav1.O
 			return err
 		}
 	}
+
 	return nil
 }
 
-func (r *OperatorPipelineReconciler) removePipelineDependencyFiles(filePath string) error {
-	if err := os.RemoveAll(filePath); err != nil {
-		log.Error(err, "Couldn't remove operator-pipelines directory")
+func (r *OperatorPipelineReconciler) applyPipelineManifests(fileName string, meta metav1.ObjectMeta, obj *tekton.Pipeline) error {
+
+	b, err := os.ReadFile(fileName)
+	if err != nil {
+		log.Error(err, fmt.Sprintf("Couldn't read manifest file for: %s", fileName))
+		return err
+	}
+
+	if err = yamlutil.Unmarshal(b, &obj); err != nil {
+		log.Error(err, "Couldn't unmarshall yaml file")
 		return err
 	}
+
+	obj.SetNamespace(meta.Namespace)
+	err = r.Get(context.Background(), types.NamespacedName{Name: obj.Name, Namespace: obj.Namespace}, obj)
+	if err != nil && errors.IsNotFound(err) {
+
+		if err = r.Client.Create(context.Background(), obj); err != nil {
+			log.Error(err, "failed to create pipeline resource")
```

Comment:
would be great to have the filename in all error messages in this function

Reply:
Done.